spec/helper.cr

```diff
@@ -11,6 +11,6 @@ end
 Spec.before_each do
   begin
     Etcd.from_env.kv.delete_prefix TEST_PREFIX
-  rescue e
+  rescue
```

Comment:
Nice little one-liner that's helpful at other times too! Allows you to just have a default return value if you don't really care about the exception.

```suggestion
  Etcd.from_env.kv.delete_prefix TEST_PREFIX rescue nil
```